README.md

```diff
@@ -46,6 +47,10 @@
 - **View the exchange rate for a particular currency**
 - **Filter and select currency**
 
+## 🚀 Live Demo <a name="live-demo"></a>
```

Comment:
- [ ] In the video where you spoke about your project, while I applaud you for speaking about the issues you faced, kindly state a piece of code you found interesting or are proud of and please ensure that when you are talking about your code, the right screen is displayed to your viewers. Currently, when talking about your code, your screen shows your live demo and not your vscode.